types/pdfmake/interfaces.d.ts

```diff
@@ -1310,9 +1310,11 @@ export interface ContentQr extends ContentBase, ForbidOtherElementProperties<"qr
      * Size in `pt` to fit the QR code in.
      *
      * As QR codes are square, this essentially sets both the width and height.
+     * pdfmake seems to shrink QR codes using certain fixed factors, so it might actually be smaller
+     * than the value given here.
```

Comment:
I'm not sure about this change.
It feels strange to comment in a type library about the functionality of the library itself. Imo, this should be part of the library documentation. I'd either change this to a statement (not "pdfmake make SEEMS to shrink", but "pdfmake does xxx") or revert this change.

Reply:
True, that was written more from a surprised/confused user's perspective 😅 

I removed the commit.